src/main/java/com/fatec/springapi4/controller/ExpertiseController.java

```diff
@@ -0,0 +1,46 @@
+package com.fatec.springapi4.controller;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.fatec.springapi4.entity.Expertise;
+import com.fatec.springapi4.service.IExpertiseService;
+
+@RestController
+@CrossOrigin
+@RequestMapping(value = "/expertise")
+public class ExpertiseController {
+
+    @Autowired
+    private IExpertiseService expServ;
```

Comment:
não precisa abreviar o nome das variáveis, fazer isso faz com que seja mais difícil de entender o código 
```suggestion
    private IExpertiseService expertiseService;
```